great_expectations/expectations/metrics/table_metrics/table_head.py

```diff
@@ -29,29 +34,40 @@ def _pandas(
         execution_engine: PandasExecutionEngine,
         metric_domain_kwargs: dict,
         metric_value_kwargs: dict,
-        metrics: Dict[str, Any],
+        metrics: dict[str, Any],
```

Comment:
Does lowercase `dict` work in python 3.7?

Reply:
My understanding is that yes, if you use `from __future__ import annotations` it works (I have been doing this for the past week or so after @Kilo59 mentioned it).